content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md

```diff
@@ -10,4 +10,59 @@ compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
 familiar with the use and operation of Istio 1.4.
 
-TBD
+# Control Plane Restructuring
+
+In Istio 1.5, we have moved towards a new deployment model for the control plane, with many components consolidated. The following describes where various functionality has been moved to.
+
+## Istiod
+
+In Istio 1.5, there will be a new deployment, `istiod`. This component is the core of the control plane, and will handle configuration and certificate distribution, sidecar injection, and more.
+
+## Sidecar Injection
+
+Previously, sidecar injection was handled by a mutating webhook that was processed by a deployment named `istio-sidecar-injector`. In Istio 1.5, the same mutating webhook remains, but it will now point to the `istiod` deployment. All injection logic remains the same.
+
+## Galley
+
+* Configuration Validation - this functionality remains the same, but is now handled by the `istiod` deployment.
+* MCP Server - the MCP server has been disabled by default. For most users, this is an implementation detail. If you do explicitly depend on this functionality, you will need to run the `istio-galley` deployment.
+* Experimental features (such as configuration analysis) - These features will require the `istio-galley` deployment.
+
+## Citadel
+
+Previously, Citadel served two functionalities: writing certificates to secrets in each namespace, and serving secrets to the `nodeagent` over `gRPC` when SDS is used. In Istio 1.5, secrets are no longer written to each namespace. Instead, they are only served over gRPC. This functionality has been moved to the `istiod` deployment.
+
+## SDS Node Agent
+
+The `nodeagent` deployment has been removed. This functionality now exists in the Sidecar
+
+## Sidecar
```

Comment:
I might re-title this subsection to be `## Envoy Proxy Certificate Rotation Without Restart`, as that is the key take-away.